src/Python/geometry/trianglemesh.cpp

```diff
@@ -69,6 +70,10 @@ void pybind_trianglemesh(py::module &m) {
             .def("purge", &geometry::TriangleMesh::Purge,
                  "Function to remove duplicated and non-manifold "
                  "vertices/triangles")
+            .def("sample_points_uniformly",
```

Comment:
Please add 
```cpp
docstring::ClassMethodDocInject(...)
```
below, for `sample_points_uniformly`, so that we hav docs the agruments.